src/main/java/com/applicaster/sport1player/Sport1PlayerAdapter.java

```diff
@@ -0,0 +1,114 @@
+package com.applicaster.sport1player;
+
+import android.app.Activity;
+
+import com.applicaster.controller.PlayerLoader;
+import com.applicaster.jwplayerplugin.JWPlayerAdapter;
+import com.applicaster.player.PlayerLoaderI;
+import com.applicaster.plugin_manager.PluginManager;
+import com.applicaster.plugin_manager.login.LoginContract;
+import com.applicaster.plugin_manager.login.LoginManager;
+import com.applicaster.plugin_manager.playersmanager.Playable;
+import com.applicaster.pluginpresenter.PluginPresenter;
+
+import java.util.Map;
+
+public class Sport1PlayerAdapter extends JWPlayerAdapter implements PresentPluginResultI {
+    private static final String PRELOAD_VOD_PLUGIN_ID = "preload_vod_plugin_id";
```

Comment:
If you can rename the variable, as this plugin will be presented not just for VODs but also for livestreams, and not only at the beggining but also in the middle of a video playing. You can call it pin_validation_plugin_id

Reply:
It was meant not only for pin validation plugin but for any plugin to run before loading the playable item.

Reply:
Got it, but it will be so custom for this client that it will be easier for the people configuring the plugin if we call it pin validation.